dev-docs/metrics.md

```diff
@@ -179,3 +179,98 @@ Strong references to instruments will be discarded when changes to the aggregate
 On the fast path the mutex is locked for the period that it takes to upgrade the weak reference. This is a fast operation, and should not block the thread for any meaningful period of time.
 
 If there is shown to be contention in future profiling we can revisit.
+
+## Adding new metrics
+There are different types of metrics.
+
+* Static - Used by us to monitor feature usage.
+* Dynamic - Defined by the user to monitor the health and performance of their system.
+
+> New features should add BOTH static and dynamic metrics.
+
+> Why are static metrics no longer recommended for users to use directly?
+> 
+> They can, but usually it'll be only a starting point for them. We can't predict the things that users will want to monitor, and if we tried we would blow up the cardinality of our metrics resulting in high costs for our users via their APMs.
+> 
+> This *is* a departure from the previous way of doing things, but as dynamic metrics are not available we need to move away from statically defined metrics.
+
+### Naming
+Metrics should be named in a way that is consistent with the rest of the metrics in the system.
+
+**Metrics**
+* `<feature>` - This should be a noun that describes the feature that the metric is monitoring.
+
+* `<feature>.<verb>` - Sub-metrics are usually a verb that describes the action that the metric is monitoring.
+
+**Attributes**
+* `<feature>.<feature-specific-attribute>` - Are always prefixed with the feature name unless they are standard metrics from the [otel semantic conventions](https://opentelemetry.io/docs/specs/semconv/general/metrics/). 
+
+### Static metrics
+When adding a new feature to the Router you must also add new static metrics to monitor the usage of that feature and users cannot turn them off.
+These metrics must be low cardinality and not leak any sensitive information. Users cannot change these metrics and they are primarily for us to see how our features are used so that we can inform future development.
+These metrics are transmitted to Apollo unless explicitly disabled.
+
+When adding new static metrics and attributes make sure to:
+* Include them in your design document.
+* Look at the [OTel semantic conventions](https://opentelemetry.io/docs/specs/semconv/general/metrics/) 
+* Notify `#proj-router-analytics` channel in Slack.
+* Add the metrics to the spreadsheet linked in the `#proj-router-analytics` channel in Slack.
+
```

Comment:
code example of what is a static metric, to be sure which is which between static and dynamic?